engine/src/imagebitmap.cpp

```diff
@@ -361,11 +361,11 @@ void MCImageBitmapPremultiplyRegion(MCImageBitmap *p_bitmap, int32_t p_sx, int32
 
 	uint8_t *t_src_ptr = (uint8_t*)p_bitmap->data + t_src_x * 4 + t_src_y * p_bitmap -> stride;
 	uint8_t *t_dst_ptr = (uint8_t*)p_pixel_ptr + t_dst_x * 4 + t_dst_y * p_pixel_stride;
-	for (uindex_t y = 0; y < t_height; y++)
+	for (uindex_t y = 0; y < (uindex_t)t_height; y++)
```

Comment:
In this case I recommend making `y` into a `index_t` rather than casting `t_height` to `uindex_t`.  `y` is never used as anything other than a counter in the `for` loop, and therefore it makes no difference whether `y` is signed or unsigned.

The same consideration applies to the `x` in the inner loop.